style.css

```diff
@@ -75,13 +77,24 @@ h3 {
   margin-left: -5px;
   border-width: 5px;
   border-style: solid;
-  border-color: grey transparent transparent transparent; 
+  border-color: grey transparent transparent transparent;
 }
 
 
 .type {
-    height: 10px;
-    width: 10px;
+    height: 25px;
+    width: 66px;
+    margin-bottom: 4px;
+    border-radius: 4px;
+    border: 1px solid;
+    color: #ffff;
```

Comment:
For readability/accessibility, some of the types would work better if the text was in black, like Fairy and Normal, but some will need white text like Fighting and Poison. It might be a good idea to create two classes one for white type and one for black type.

Reply:
Done, let me know what you think :)

Reply:
Awesome, I think bug could maybe be black as well and fighting maybe would be better as white 🤔 This is a good resource for checking the accessibility of colours with background and foreground - https://webaim.org/resources/contrastchecker/ getting it to at least AA standards is always good. 

You can mess around with that but apart from that the code itself looks good!

Reply:
Updated all font/background colours so they meet AAA standard on https://webaim.org/resources/contrastchecker/

Reply:
Amazing! 👏 🏆